ecmd-core/cmd/help/setconfig.htxt

```diff
@@ -14,6 +14,11 @@ Syntax: setconfig [<Chip[.ChipUnit]>] <ConfigName> <Value> [-i<format>] [-dk | -
                         with ChipUnit: pu.mc
 
         ConfigName      Name of configuration variable to write
+	
+	Mode [opt]	To be used only with configuration variables that support multiple values. 
+	     		Mode can either be "add" or "remove" to add or remove the specified value 
+			from the configuration variable. If Mode is not specified, Value will 
```

Comment:
Does the alignment of this text and the example text below look okay when you `cat` it out to the screen?  That's the important thing.  Maybe the alignment here is just an artifact of the GUI view.

Reply:
Actually, you probably don't want to commit the help text update until you do the ecmdMiscUser.C updates.